src/EditorFeatures/Core/Shared/Tagging/EventSources/TaggerConstants.cs

```diff
@@ -19,21 +19,26 @@ internal static int ComputeTimeDelay(this TaggerDelay behavior, ITextBuffer text
             {
                 // TODO : should we remove TaggerBehavior enum all together and put NearImmediateDelay
                 // const in Interaction?
-                switch (behavior)
-                {
-                    case TaggerDelay.NearImmediate:
-                        return NearImmediateDelay;
-                    case TaggerDelay.Short:
-                        return ShortDelay;
-                    case TaggerDelay.Medium:
-                        return MediumDelay;
-                    case TaggerDelay.OnIdle:
-                    default:
-                        return IdleDelay;
-                }
+                return ToTimeDelay(behavior);
             }
 
             return NonFocusDelay;
         }
+
+        public static int ToTimeDelay(this TaggerDelay behavior)
```

Comment:
Can this return something stronger than an 'int'? I'll assume the delay is millennia otherwise.